// Copyright 2015 Google Inc. All Rights Reserved.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
// License for the specific language governing permissions and limitations under
// the License.

syntax = "proto2";


import "cell.proto";

package dsrf;


message Row {

  // The type of the row in the spreadsheet (eg. "MW01").
  optional string type = 1;

  // The line number in the report file.
  optional uint32 row_number = 2;

  // The cell objects in the row. The objects are in the right order, as they
  // appear in the file.
  repeated Cell cells = 3;
}
